scijava-ops-engine/src/main/java/org/scijava/ops/engine/util/internal/OpMethodUtils.java

```diff
@@ -108,4 +116,30 @@ public static Type[] getOpParamTypes(
 			.toArray(Type[]::new);
 	}
 
+	public static StructInstance<?> createOpInstance(final OpInfo info,
```

Comment:
Could use javadoc since this is a public utility method, especially the `org.scijava.ops.engine` module readability limitation

Reply:
@hinerm it's only public in that it can be read from other packages/classes, note that we do not `export`/`open` this package.

But sure, not a terrible idea to document